xbmc/games/addons/GameClientMouse.h

```diff
@@ -21,6 +21,8 @@
 
 #include "input/mouse/IMouseInputHandler.h"
 
+#define MOUSE_CONTROLLER_ID "game.controller.mouse"
```

Comment:
Kind of like how DEFAULT_CONTROLLER_ID is done, but it'd be nice if we could keep this #define inside MouseWindowingButtonMap.cpp and access it through its ControllerID() function